osquery/core/init.cpp

```diff
@@ -291,6 +290,13 @@ Initializer::Initializer(int& argc,
   // Let gflags parse the non-help options/flags.
   GFLAGS_NAMESPACE::ParseCommandLineFlags(argc_, argv_, isShell());
 
+  if (isShell()) {
+    // Do not set these values before calling ParseCommandLineFlags.
```

Comment:
```suggestion
    // Set these values after calling ParseCommandLineFlags.
```

Reply:
It's a good suggestion but I don't feel like it's really a double negative. I prefer to have the "Do not" as a more powerful hint to anyone making changes.